package.json

```diff
@@ -17,14 +17,16 @@
   "lint-staged": {
     "*.{js,ts,tsx}": [
       "npm run eslint:base -- --fix",
-      "npm run prettier:base -- --write"
+      "npm run prettier:base -- --write",
+      "npm run tsc:check"
     ],
     "*.{json,md,mdx}": [
       "npm run prettier:base -- --write"
     ]
   },
   "scripts": {
     "build": "lerna run build",
+    "tsc:check": "lerna run tsc:check && echo",
```

Comment:
In case anyone wonders about echo: https://stackoverflow.com/a/73007124

The alternative for that is removing lint-staged from package.json and move configuration to a separate lint-stages.config.js nad there we can use functions as:
```
module.exports = {
"*.{js,ts,tsx}": [() => "npm run tsc:check"]
}
```